src/components/item/index.tsx

```diff
@@ -55,7 +60,10 @@ const Product = ({
         <div className="card-thumbnail">
           {imageString ? (
             <Anchor path={`/item/${itemId}`}>
-              <Image src={imageString} alt="NFT_portfolio" width={533} height={533} />
+              <img
```

Comment:
Might be a better aproach use the Image component from Next and use the ALT prop to improve the SEO.
something like this: **alt={``${title}#${tokenId} - Leda MarketPlace. ${authors}``}** or whatever

Reply:
Unless there is no way to avoid the visual issue, I agree to use img. As Julian suggested, it is better to rely on the Image component that nextjs has OOB

Reply:
Done!